index.html

```diff
@@ -33,28 +33,212 @@ <h1 class="headline">
+                            <div class="tech-item">
+                                Ruby on rails
+                            </div>
+                            <div class="tech-item">
+                                css
+                            </div>
+                            <div class="tech-item">
+                                JavScript
+                            </div>
+                            <div class="tech-item">
+                                html
+                            </div>
+                        </div>
+                        <button type="button" class="see-project-btn">See Project</button>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <section class="center-content">
+            <h1 class="headline about-me">About me</h1>
+            <p class="supporting-text">
+                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.
+            </p>
+            <button type="button" class="see-project-btn">
+                Get My Resume
+            </button>
 
+            <ul class="about-me-categories">
+                <li class="about-me-card">
+                    <img class="icon" src="images/Icons.png" alt="language-icon">
+                    <h3 class="about-me-item">Languages</h3>
+                    <div class="about-me-tags">
+                        <div class="tag">Javascript</div>
+                        <div class="tag">Ruby</div>
+                        <div class="tag">Html</div>
+                        <div class="tag">CSS</div>
+                    </div>
+                </li>
+
+                <li class="about-me-card">
+                    <img class="icon" src="images/Icons-framework.png" alt="framework-icon">
+                    <h3 class="about-me-item">Frameworks</h3>
+                    <div class="about-me-tags">
+                        <div class="tag">Bootstrap</div>
+                        <div class="tag">Ruby on rails</div>
+                        <div class="tag">RSpec</div>
+                        <div class="tag">Capybara</div>
+                        <div class="tag">Selenium</div>
+                    </div>
+                </li>
+
+                <li class="about-me-card">
+                    <img class="icon" src="images/Icons-skills.png" alt="Skills icon">
+                    <h3 class="about-me-item">Skills</h3>
+                    <div class="about-me-tags">
+                        <div class="tag">Codekit</div>
+                        <div class="tag">Github</div>
+                        <div class="tag">Codepen</div>
+                        <div class="tag">Gitlab</div>
+                        <div class="tag">Terminal</div>
+                    </div>
+                </li>
+
+            </ul>
```

Comment:
Kindly change the `div` that takes class `about-me-tags` to be `ul` tag contains multiple `li` tags instead of `div` tag contains multiple `div` tags